tests/test_fixtures/rule_based_profiler/example_notebooks/BasicExample_RBP_Instantiation_and_running.ipynb

```diff
@@ -758,6 +758,361 @@
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from great_expectations.rule_based_profiler.domain_builder import ColumnDomainBuilder"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 29,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "domain_builder: DomainBuilder = ColumnDomainBuilder(\n",
+    "    data_context=data_context,\n",
+    "    batch_request=\"$variables.multi_batch_batch_request\",\n",
+    "    include_column_names=[\"tip_amount\", \"fare_amount\"]\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Instantiating `ParameterBuilders`\n",
+    "\n",
+    "Our Rule will contain 2 `NumericMetricRangeMultiBatchParameterBuilders`, one for each of our 2 Expectation types. One will be estimating the Parameter values for the `column.min` Metric, and the other will be estimating Parameter values for the `column.max` Metric.  `metric_domain_kwargs` are passed in from our DomainBuilder using `$domain.domain_kwargs`.\n",
+    "\n",
+    "Also note the use of 3 Variables we defined above: \n",
+    "\n",
+    "- `$variables.estimator_name`: This is `\"oneshot\"` in our case. \n",
+    "- `$variables.false_positive_rate`: This is `1.0e-2` or 1% in our case.\n",
+    "- `$variables.multi_batch_batch_request`: This the `multi_batch_batch_request`, which gives all 12 Batches of data from the 2018 taxi_data datataset."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 30,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from great_expectations.rule_based_profiler.parameter_builder import NumericMetricRangeMultiBatchParameterBuilder"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 31,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "min_range_parameter_builder: NumericMetricRangeMultiBatchParameterBuilder = NumericMetricRangeMultiBatchParameterBuilder(\n",
+    "    name=\"min_range_parameter_builder\",\n",
+    "    metric_name=\"column.min\",\n",
+    "    estimator=\"$variables.estimator_name\",\n",
+    "    metric_domain_kwargs=\"$domain.domain_kwargs\",\n",
+    "    false_positive_rate='$variables.false_positive_rate',\n",
+    "    data_context=data_context,\n",
+    "    batch_request=\"$variables.multi_batch_batch_request\",\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 32,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "max_range_parameter_builder: NumericMetricRangeMultiBatchParameterBuilder = NumericMetricRangeMultiBatchParameterBuilder(\n",
+    "    name=\"max_range_parameter_builder\",\n",
+    "    metric_name=\"column.max\",\n",
+    "    estimator=\"$variables.estimator_name\",\n",
+    "    metric_domain_kwargs=\"$domain.domain_kwargs\",\n",
+    "    false_positive_rate=\"$variables.false_positive_rate\",\n",
+    "    data_context=data_context,\n",
+    "    batch_request=\"$variables.multi_batch_batch_request\",\n",
```

Comment:
@Shinnnyshinshin Any chance to keep the order of arguments the way they are in the class definition (for elegance/readability purposes, that is) -- thanks!